packages/moonstone/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The following is a curated list of changes in the Enact moonstone module, newest
 ### Added
 
 - `moonstone/VirtualList.VirtualList` and `moonstone/VirtualList.VirtualGridList` `role="list"`
+- `moonstone/Panels` prop `closeButtonAriaLabel` to configure the aria-label set on application close button
+- `moonstone/Picker` props `aria-label` , `decrementAriaLabel`, `incrementAriaLabel` to configure the label set on each button
```

Comment:
please add `and` between `, `incrementAriaLabel``
Also for DatePicker and TimePicker

Reply:
I added `and` between last `PROP`